poc/tests/test_idpf_bbcggi21.py

```diff
@@ -181,3 +331,18 @@ def test_public_share_roundtrip(self) -> None:
             idpf.decode_public_share(idpf.encode_public_share(public_share)),
             public_share,
         )
+
+
+def bytes_to_bit_string(s: bytes) -> tuple[bool, ...]:
```

Comment:
Wouldn't this be the same procedure used by the agg param decoder?

Reply:
This is similar, but simpler since it only produces one bit string, and is restricted to only handling multiples of eight bits.